src/mindist.rs

```diff
@@ -203,7 +203,7 @@ fn D_rk(r: usize, k: usize, bez1: &[Vec2], bez2: &[Vec2]) -> f64 {
 
 // Bezier basis function
 fn basis_function(n: usize, i: usize, u: f64) -> f64 {
-    choose(n, i) as f64 * (1.0 - u as f64).powi((n - i) as i32) * u.powi(i as i32)
+    choose(n, i) as f64 * (1.0 - u as f64).powi(n as i32 - i as i32) * u.powi(i as i32)
```

Comment:
What expectation do we have for inputs, and what is the desired behaviour? are we happy to see negative powers here?

Reply:
These are fed from a loop which ranges over 0..order_of_bezier

Reply:
So i >n is possible? That really shouldn't be, it would also be a divide by zero when u is 0 or 1. Can you explain how this might happen?

Reply:
I think this was fixing the symptom, whereas the next commit actually fixed the disease.